### PR TITLE
N-ary (general) tree

### PR DESCRIPTION
**Pull request by Aarti002**
issue #694 solved
python

### CODE REVIEW

**Comment by Aarti002** [comment]
hello @sayantani11 , i think i have made required changes can you please review my code again
Thank you!

**Review by sayantani11 — APPROVED** [review]

**Comment by Rohan27s** [comment]
@Aarti002 Update the Read me first

**Comment by Aarti002** [comment]
@Rohan27s i updated readme file but i don't know why it shows conflicts everytime i do it?

File: Code/Python/n_ary_tree.py [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+               w r
+
+
+
+  """
+#Time complexity: O(n)          ->number of nodes
+#Space complexity: O(n)         ->number of nodes
+
+
+
```
Comment (Rohan27s) [inline Code/Python/n_ary_tree.py:140]: Remove the unnecessary spaces from the whole code

File: Code/Python/n_ary_tree.py [hunk 2]
```diff
@@ -0,0 +1,140 @@
+# Ques: Create a general (N-ary) tree
+# Language: Python
+
+# Firstly, class treenode is defined with several inner functions including "__init__" for creating tree node with chils as list and #intitially storing 'None' as address of parent, "add_child" add data in child list, there is a helping function named "get_level" as #well to get the level of a child node while printing its value and finally "print_tree" function prints tree with according to its #level .
```
Comment (Rohan27s) [inline Code/Python/n_ary_tree.py:4]: Use multi-line comment here and remove the space

**Review by Rohan27s — CHANGES_REQUESTED** [review]

**Review by Rohan27s — APPROVED** [review]

**Pull request merged**